subcommands/config/log.go

```diff
@@ -37,7 +45,8 @@ func doConfigLog(cmd *cobra.Command, args []string) {
 	} else {
 		logrus.Debugf("Showing config history for %s group %s", factory, group)
 		subcommands.LogConfigs(&subcommands.LogConfigsOptions{
-			Limit: listLimit,
+			UserLookup: lookups,
+			Limit:      listLimit,
```

Comment:
A nit: can you move the `UserLookup` to the end of the struct args (here and above)?

Reply:
Only this one, but you can ignore it.

Reply:
it was a failed force-push. the new one has what you wanted